pages.zh/linux/boltctl.md

```diff
@@ -0,0 +1,28 @@
+# boltctl
+
+> 控制雷电 (thunderbolt) 设备。
+> 更多信息：<https://manpages.debian.org/latest/bolt/boltctl.1.html>.
```

Comment:
Any reason why you didn't use the Chinese period here?  That's what I'd like to do for all of the pages (#5240).
```suggestion
> 更多信息：<https://manpages.debian.org/latest/bolt/boltctl.1.html>。
```

Reply:
Also, I see there's no space after Chinese colon.  Is that considered correct? Or is it technically correct to have a space after it, even though it's a full-width character, like this?
```
> 更多信息： <https://manpages.debian.org/latest/bolt/boltctl.1.html>.
```

Reply:
> Also, I see there's no space after Chinese colon. Is that considered correct? Or is it technically correct to have a space after it, even though it's a full-width character, like this?
> 
> ```
> > 更多信息： <https://manpages.debian.org/latest/bolt/boltctl.1.html>.
> ```

We don't add a space after the full-width colon.

Reply:
Well, I'd like to do the most proper and high-quality thing here.  If you don't mind, please state your opinions in #5240 for reference and history.